### PR TITLE
Complete diagnostic on jquery-dom

### PR DESCRIPTION
**Pull request by sujkid**
Diagnostic for the day complete

### CODE REVIEW

**Comment by sujkid** [comment]
Comfort: 3
Clarity: 3

I used google

File: diagnostic.md [hunk 1]
```diff
@@ -3,67 +3,71 @@
...
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('ol').children();
+$('li').siblings();
```
Comment (neugierige) [inline diagnostic.md:19]: Your 2nd answer is not incorrect, but you could actually just do `$('li')`

File: diagnostic.md [hunk 2]
```diff
@@ -3,67 +3,71 @@
...
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('.listerItem').parentUtil('form');
```
Comment (neugierige) [inline diagnostic.md:26]: 1) you spelled `parentsUntil` wrong... https://api.jquery.com/parentsUntil/
2) `form` is the child of body, which is not a direct parent of `.listerItem`. 

`$('.listerItem').parent().parent().find('form');` OR `$('.listerItem').parent().parent().children('form');`

File: diagnostic.md [hunk 3]
```diff
@@ -3,67 +3,71 @@
...
 
 ## Question 4
 
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+Getter methods -
+.css() - Returns the css property of the selected node.
```
Comment (neugierige) [inline diagnostic.md:34]: yes, but you need more than that. `.css('<some property>')`

File: diagnostic.md [hunk 4]
```diff
@@ -3,67 +3,71 @@
...
+.html() - Returns the html tag of the selected node.
+
+Setter methods -
+.text() - When passed a parameter, it sets the parameter text to the selected
+          node.
+.css() - When passed a parameter, it sets the parameter css property to the
+         selected node.
```
Comment (neugierige) [inline diagnostic.md:41]: for setters, you need to specify _what_ you want to set

```
.text('some text')
.css('<some property>', '<some value>')
```

File: diagnostic.md [hunk 5]
```diff
@@ -3,67 +3,71 @@
...
 
-<!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
+DOM, or Document Object Model is a visual representation of the HTML page.
+It is useful for accessing and manupulating the nodes using javascript.
```
Comment (neugierige) [inline diagnostic.md:49]: It's not a visual representation, although we like to visualize it with a tree-like diagram. Also, the DOM is represented in the browser by the `document` object.

File: diagnostic.md [hunk 6]
```diff
@@ -3,67 +3,71 @@
...
 
 ## Question 7
 
 Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('li').parents().hide();
```
Comment (neugierige) [inline diagnostic.md:65]: So close! `$('li').parent().parent().hide();`

Note that `parent` !== `parents`

File: diagnostic.md [hunk 7]
```diff
@@ -3,67 +3,71 @@
...
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
-<!-- your answer starts here -->
+$('div').on(click, function() {
 
-<!-- your answer ends here -->
+});
```
Comment (neugierige) [inline diagnostic.md:73]: Fine, but note that you need `''` around `click`
`$('div').on('click', function() {
});`

**Comment by neugierige** [comment]
3/8. Hi Suja, all this new terminology and concepts will become familiar once you start using them. But I would just review the lesson again, and make sure you understand why the answers are what they are.

**Pull request closed without merging**